packages/ava/src/data/statistics/stdlib/cdf.ts

```diff
@@ -0,0 +1,20 @@
+import { isNil } from 'lodash';
+
+/**
+ * Evaluates the cumulative distribution function (CDF) for a normal distribution at a value x
+ * - Reference to equation 26.2.17 in Abramowitz and Stegun: Handbook of Mathematical Functions
+ * @param mu mean
+ * @param sigma standard deviation
+ * */
+export const cdf = (x: number, mu: number = 0, sigma: number = 1): number => {
+  if (sigma < 0 || [x, mu, sigma].some((value) => isNil(value))) return NaN;
+  // transfer to standard normal distribution
+  const normalX = (x - mu) / sigma;
+  /** probability density function of the standard normal distribution */
+  const Zx = (1 / Math.sqrt(2 * Math.PI)) * Math.exp((-1 * normalX ** 2) / 2);
```

Comment:
the name `Zx` is not standard, but necessary?

Reply:
I just want it to fit the semantics of the formula. Is it necessary to modify to a standard name?
![image](https://user-images.githubusercontent.com/71261480/224596832-72f7a066-a863-4ffe-8a88-c35299ebcf36.png)

Reply:
okay, keep it. It's a special situation.